packages/morph/src/dom.js

```diff
@@ -14,12 +14,18 @@ export let dom = {
     replace(children, old, replacement) {
         let index = children.indexOf(old)
 
+        let replacementIndex = children.indexOf(old)
```

Comment:
Heads up, this should be "replacement" instead of "old" - tracked it down because of a pagination test failing. Fixed now. Thanks!